### PR TITLE
Connor/chat screen

### PR DESCRIPTION
**Pull request by ConnorBritton**

### CODE REVIEW

File: public/client.js [hunk 1]
```diff
@@ -97,7 +97,19 @@ $(function () {
       $('#chatApprovalButtons').show();
     }
   })  
-  
 
+  // Chat Logic
+  $('#enterChatButton').click(function(){
```
Comment (farfar336) [inline public/client.js:102]: Note: This was already implemented in the Lobby screen back end. I'll need to resolve merge conflicts for this

File: public/client.js [hunk 2]
```diff
@@ -97,7 +97,19 @@ $(function () {
...
+    $('#lobby').hide();
+    $('#chat').css('display', 'contents');
+  })
+
+  $('#messageForm').submit(function(e){
```
Comment (farfar336) [inline public/client.js:107]: What does this part of the code do?

**Comment by farfar336** [comment]
I tested the application and it works as intended.

1 comment needs to be addressed in the code before merging

File: public/client.js [hunk 3]
```diff
@@ -97,7 +97,19 @@ $(function () {
       $('#chatApprovalButtons').show();
     }
   })  
-  
 
+  // Chat Logic
+  $('#enterChatButton').click(function(){
+    $('#lobby').hide();
+    $('#chat').css('display', 'contents');
```
Comment (farfar336) [inline public/client.js:104]: Is it possible to replace line 104 with this:

$('#chat').show();

Comment (ConnorBritton) [reply public/client.js:104]: I originally had that but $('#chat').show(); sets 'display' to 'block' so I looked it up and it said to do what I did on stack overflow.
image

Comment (ConnorBritton) [reply public/client.js:104]: https://stackoverflow.com/questions/9260009/how-to-add-displayinline-block-in-a-jquery-show-function

Comment (ConnorBritton) [reply public/client.js:104]: And block doesn't work for my ui for some reason

**Pull request merged**